src/methods/modular_programs/medium/q3/AlternateIt.java

```diff
@@ -0,0 +1,29 @@
+package methods.modular_programs.medium.q3;
+
+/**
+ * *Write a Java program with a method called sumCalc to ask the user for a
```

Comment:
Please specify what parts of the question are to be done in the main method and what parts of the question are to be done in the sumCalc method. I think it is unclear currently.

Reply:
You also do not specify the parameters and the return type of the method.